charts/aws-cloud-controller-manager/values.yaml

```diff
@@ -0,0 +1,116 @@
+namespace: "kube-system"
+
+extraArgs: {}
```

Comment:
this doesn't appear to be used

Reply:
@wongma7 This is correct, but expecting optional flags to be added in the future. So just kept with null values. I have few other values that are also added which are commented on. If you want to remove or wanted to add as a comment. Please let me know. I am okay with either of these changes.

Reply:
okay, i'm fine with keeping it as placeholder

Reply:
Okay Thanks, @wongma7. I am resolving this review comment